<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PCAMainInfoWidget</class>
 <widget class="QWidget" name="PCAMainInfoWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>150</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="labelPop">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    font-weight:bold;
    border-style:solid;
    border-left-width:4px;
    border-color:#2f3640;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>Populations</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelSampleOut">
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelPopOut">
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelSample">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    font-weight:bold;
    border-style:solid;
    border-left-width:4px;
    border-color:#2f3640;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>Samples</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelPC2Out">
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QWidget" name="nullWidget1" native="true"/>
      </item>
      <item row="5" column="1">
       <widget class="QWidget" name="nullWidget2" native="true"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelPC1">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    font-weight:bold;
    border-style:solid;
    border-left-width:4px;
    border-color:#2f3640;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>PC1</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelPC1Out">
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelPC2">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    font-weight:bold;
    border-style:solid;
    border-left-width:4px;
    border-color:#2f3640;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>PC2</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelPC3">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    font-weight:bold;
    border-style:solid;
    border-left-width:4px;
    border-color:#2f3640;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string>PC3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelPC3Out">
        <property name="styleSheet">
         <string notr="true">QLabel{
    font-family:&quot;Times New Roman&quot;;
    background-color:transparent;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
